actions/update-deps/action.yml

```diff
@@ -65,10 +65,10 @@ runs:
       shell: bash
       run: |
         if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-          echo "branch=buddy-${{ github.event.pull_request.number }}" >> $GITHUB_OUTPUT
-          echo "target=update-buddy-version" >> $GITHUB_OUTPUT
+          echo "branch=${{ github.action_repository }}-${{ github.event.pull_request.number }}" >> $GITHUB_OUTPUT
+          echo "target=update-${{ github.action_repository }}-version" >> $GITHUB_OUTPUT
```

Comment:
So it's going to be `update-manticoresoftware/manticoresearch-buddy-version` ? Is it on purpose?